README.md

```diff
@@ -5594,10 +5518,13 @@ most secure format supported by the most platforms.
 
 This module is CI tested against both [open source Puppet][] and [Puppet Enterprise][] on:
 
-- CentOS 5 and 6
-- Ubuntu 12.04 and 14.04
-- Debian 7
-- RHEL 5, 6, and 7
+- CentOS 7
+- Ubuntu 14.04 and 16.04
+- Debian 8
+- RHEL 7
+- Oracle 7
+- Scientific 7
+- SLES 12
```

Comment:
We have started removing mentions of supported OS from the readme. Maybe just replace with a direction towards the metadata.

Reply:
Will remove post README review.